### ISSUE TITLE
How Did You Create Wurst?

### ISSUE THREAD

**Issue by Orcunegeacar** [Wurst-Imperium/Wurst-Client#510]
I want to know. How did you create Wurst? In Eclipse? I know you did that in java. How did this idea come to your brain. How did you find that idea. What was the first release. I think you should do a new version of first release for Wurst. Or a film. How did the Wurst come. A Trailer maybe :smile: I guess I wanted a lot of things :laughing:

**Comment by pokeball99** [comment]
Error:bufferoverflow :floppy_disk:

**Comment by TekkitCommando** [comment]
Dang thats a lot of questions

**Comment by pokeball99** [comment]
Yea that's y I made that joke :D

**Comment by TekkitCommando** [comment]
:+1: :P

**Comment by Alexander01998** [comment]
> I want to know. How did you create Wurst?

I started it based on [ConnorM's tutorial](http://www.wizardhax.com/how-to-code-a-minecraft-hacked-client/), without having any experience with Java. After episode 8, he stopped making tutorials for a while and I didn't want to wait, so I watched some other tutorials, read through the HackForums and MPGH, and tried to make some mods on my own. I gained a lot of experience by doing that, and by the time Connor continued his tutorial series, I didn't need it anymore...

> In Eclipse?

Yes.

> How did this idea come to your brain. How did you find that idea.

I started using hack clients after @gruenwald32 told me that Nodus could bypass WorldGuard (which turned out to be false). I then downloaded some other clients and couldn't find one that really fits my needs (although WeepCraft was pretty good), so decided to make my own client.

> What was the first release.

The first release was Wurst v1.0 Beta, but it was never released. I don't remember what happened exactly, but I somehow lost it and released v1.1 Beta instead. Both of these were Closed Beta releases, so I only released them to the Wurst-Imperium. The first Public Beta release was Wurst v1.8 Beta.

All of the Closed Beta & Public Beta releases are still available on the [Wurst Client website](https://www.wurst-client.tk/downloads/). Just click on the "Looking for the old versions?" link.

> I think you should do a new version of first release for Wurst.

Not needed because it is still available and not possible because I no longer have the source code for it.

> Or a film.

That is planned for the first anniversary of Wurst (May 1).

> How did the Wurst come. A Trailer maybe

I guess I have already answered that above.
